tests/test_diagnostics.py

```
@@ -36,11 +36,22 @@ class _OpaqueDiagnosticValue:
+            memo (dict[int, Any]): The memo argument.
+        """
         return self
 
     def __repr__(self) -> str:
-        """Fail if diagnostics attempts to render the private object."""
+        """Fail if diagnostics attempts to render the private object.
+
+        Returns:
+            The returned value.
+        """
         raise AssertionError("opaque diagnostics values must not be rendered")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)